tools/diagnostics/bundle.py

```diff
@@ -0,0 +1,21 @@
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+
+class Bundle(object):
```

Comment:
Do you need the `(object)` in Python3?

Reply:
7e8bf60cded2dc95da43e0adc5ff19d9a11f2d43